content/en/docs/concepts/services-networking/service.md

```diff
@@ -642,9 +642,9 @@ You must enable the `ServiceLBNodePortControl` feature gate to use this field.
 
 #### Specifying class of load balancer implementation {#load-balancer-class}
 
-{{< feature-state for_k8s_version="v1.21" state="alpha" >}}
+{{< feature-state for_k8s_version="v1.22" state="beta" >}}
 
-Starting in v1.21, you can optionally specify the class of a load balancer implementation for
+From Kubernetes v1.21 onwards, you can optionally specify the class of a load balancer implementation for
```

Comment:
I think this should say v1.22 because that's when it's on by default

Reply:
We should say that it's available from v1.21 and also say that the feature gate is enabled by default from v1.22 onwards; that'll match how the rest of the docs tackle this sort of thing.